libs/compute_api/src/spec.rs

```diff
@@ -144,8 +144,9 @@ impl RemoteExtSpec {
             // sometimes library names might have a suffix like
             // library.so or library.so.3. We strip this off
             // because library_index is based on the name without the file extension
-            let strip_lib_suffix = Regex::new(r"\.so.*").unwrap();
-            let lib_raw_name = strip_lib_suffix.replace(real_ext_name, "").to_string();
+            let strip_lib = Regex::new(r"([^\/]+)\.so.*").unwrap();
```

Comment:
Can't we just use this API: https://doc.rust-lang.org/std/path/struct.Path.html#method.file_name